amplify/backend/api/colonycdapp/schema/schema.graphql

```diff
@@ -8,6 +8,24 @@ input AMPLIFY {
 # https://docs.amplify.aws/cli-legacy/graphql-transformer/overview/#api-category-project-structure
 # but I never could get it to work
 
+"""
+Input data for fetching domain balance in/out values for a specific timeframe
+"""
+input DomainBalanceArguments {
+  """
+  Address of the colony on the blockchain
+  """
+  colonyAddress: String!
+  """
+  Address of the domain on the blockchain
+  """
+  domainAddress: String!
```

Comment:
I think it's best to add a short descriptor comment for each field in this type (and all the other new types) to keep style with the rest of this file, where everything has a comment. It helps to understand every detail of the schema in the future, even when it seems obvious what something is :)